webrender/src/clip.rs

```diff
@@ -109,18 +109,16 @@ impl From<ClipRegion> for ClipSources {
 
 #[derive(Debug)]
 pub struct ClipSources {
-    clips: Vec<ClipSource>,
-    pub mask_cache_info: MaskCacheInfo,
+    pub clips: Vec<ClipSource>,
+    pub handles: Vec<GpuCacheHandle>,
```

Comment:
why don't we want to either simply include `GpuCacheHandle` into `ClipSource` or have it stored nearby, e.g. `Vec<(ClipSrouce, GpuCacheHandle)>`?

Reply:
I did consider including it in ```ClipSource``` but it seemed a bit cleaner conceptually to separate them. No real reason it couldn't be in a tuple though!

Reply:
Great, let's tuple it up then ;)